### PR TITLE
Updated instructions for using blueflood plugin with grafana

### PR DESCRIPTION
**Pull request by ChandraAddala**

### CODE REVIEW

**Comment by ChandraAddala** [comment]
@goru97 Pls review.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,73 @@
...
+## Steps to build plugin from source code
+
+```
+cd blueflood-grafana
 npm install</br>
 npm install -g grunt-cli</br>
 grunt</br>
+```
+
+The built plugin is available in `./dist` directory. Install the plugin as shown below and restart grafana.
```
Comment (goru97) [inline README.md]: Replace `Install the plugin as shown below` with `Copy dist to grafana plugins direcory`.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,73 @@
...
+
+The built plugin is available in `./dist` directory. Install the plugin as shown below and restart grafana. 
+ 
+```
 cp -r ./dist /var/lib/grafana/plugins
+```
+
+##Configure Grafana
+
+Follow the step by step [instructions](https://github.com/rackerlabs/blueflood-grafana/wiki/Getting-started) to configure
```
Comment (goru97) [inline README.md]: remove `step by step`

File: README.md [hunk 3]
```diff
@@ -1,10 +1,73 @@
...
+
+##Configure Grafana
+
+Follow the step by step [instructions](https://github.com/rackerlabs/blueflood-grafana/wiki/Getting-started) to configure 
+blueflood or Rackspace-metrics as datasource.
+
+## Blueflood datasource
+
+[Blueflood](https://github.com/rackerlabs/blueflood) is an open source multi-tenant, distributed metric processing 
+system. This plugin allows you to visualize metrics stored in blueflood thru grafana.
```
Comment (goru97) [inline README.md]: replace `thru` with `through / via`

File: README.md [hunk 4]
```diff
@@ -1,10 +1,73 @@
...
+
+## Blueflood datasource
+
+[Blueflood](https://github.com/rackerlabs/blueflood) is an open source multi-tenant, distributed metric processing 
+system. This plugin allows you to visualize metrics stored in blueflood thru grafana. 
+
+## Rackspace-metrics datasource
 
+Rackspace-metrics is a SaaS offering for storing metrics. This plugin allows you to visualize metrics stored in 
+Rackspace-metrics thru grafana.
```
Comment (goru97) [inline README.md]: Same replace `thru` with `via`

**Comment by ChandraAddala** [comment]
@goru97 Updated as per your comments.

File: README.md [hunk 5]
```diff
@@ -1,10 +1,73 @@
...
+
+##Configure Grafana
+
+Follow the [instructions](https://github.com/rackerlabs/blueflood-grafana/wiki/Getting-started) to configure 
+blueflood or Rackspace-metrics as datasource.
+
+## Blueflood datasource
+
+[Blueflood](https://github.com/rackerlabs/blueflood) is an open source multi-tenant, distributed metric processing 
+system. This plugin allows you to visualize metrics stored in blueflood though grafana.
```
Comment (goru97) [inline README.md]: Replace `though` with `through` or `via` is even better.

File: README.md [hunk 6]
```diff
@@ -1,10 +1,73 @@
...
+
+## Blueflood datasource
+
+[Blueflood](https://github.com/rackerlabs/blueflood) is an open source multi-tenant, distributed metric processing 
+system. This plugin allows you to visualize metrics stored in blueflood though grafana. 
+
+## Rackspace-metrics datasource
 
+Rackspace-metrics is a SaaS offering for storing metrics. This plugin allows you to visualize metrics stored in 
+Rackspace-metrics though grafana.
```
Comment (goru97) [inline README.md]: Same

File: README.md [hunk 7]
```diff
@@ -1,10 +1,73 @@
...
+        rackerlabs/graphite-api-blueflood-finder
+```  
+
+* A grafana>=3.0 installation 
+
+
+## Steps to install plugin 
+
+You can build the plugin from source code(described in next section) or you can directly use the transpiled code, 
+built from this source code, which is maintained in a [separate repository](https://github.com/ChandraAddala/blueflood-grafana-plugin).
```
Comment (goru97) [inline README.md]: Why is it pointing to your repo when the plugin is there at `rackerlabs`?

File: README.md [hunk 8]
```diff
@@ -1,10 +1,73 @@
...
+# graphite-api with blueflood finder
+    docker run \
+        -d -p 8888:8888 \
+        -e GRAFANA_URLS=http://localhost:3000,http://192.168.1.200:3000 \
+        -e BLUEFLOOD_QUERY_URL=http://localhost:20000\
+        -e TENANT_ID=123 \
+        rackerlabs/graphite-api-blueflood-finder
+```  
+
+* A grafana>=3.0 installation
```
Comment (goru97) [inline README.md]: replace `grafana>=3.0` with grafana 3.x. We don't know whether this will work with grafana 4.x?

File: README.md [hunk 9]
```diff
@@ -1,10 +1,73 @@
-# grafana_blueflood_3.0.0
-Blueflood Plugin source code for Grafana (Using Graphite - API).
+# blueflood-grafana
 
-## Steps to install plugin
-cd blueflood-grafana-graphite_api-plugin-src
+Source code of the blueflood-grafana plugin to use [Blueflood](https://github.com/rackerlabs/blueflood) or 
+Rackspace-metrics as a datasouce with [grafana](http://grafana.org/). This plugin works with Grafana 3.0 and higher.
```
Comment (goru97) [inline README.md]: replace `Grafana 3.0 and higher` with `Grafana 3.x`

**Comment by ChandraAddala** [comment]
@goru97 I had addressed your comments. Pls take a look.

**Comment by goru97** [comment]
:thumbsup:

**Pull request merged**